### DISCUSSION TITLE
Changing the title bars for ableton and reaper?

### DISCUSSION THREAD

**Discussion by Tasenova99** [MicaForEveryone/MicaForEveryone#338] [General]
I have mica installed, 
and these 
NET Core 3.1 |
• Visual C++ Runtime |

But the title bar is still white in both applications. I am trying to get these white strips gone as it distracts my eyes. 
![sdfsdfdsfd](https://github.com/MicaForEveryone/MicaForEveryone/assets/94333547/b8e075e0-d71a-438d-8a2c-056dd0df1c0f)

**Comment by dongle-the-gadget** [comment]
That's the menu bar, which we can't change at this moment.
